var/spack/repos/builtin/packages/py-pytest/package.py

```diff
@@ -53,7 +61,8 @@ class PyPytest(PythonPackage):
         depends_on("py-setuptools-scm", when="@3.1:")
 
     with default_args(type=("build", "run")):
-        depends_on("python@3.8:", when="@8:")
+        depends_on("python@3.9:", when="@8.3.4:")
```

Comment:
Where do you see this? I still see 3.8+ in `pyproject.toml`

Reply:
I mistakenly pulled the version from main and not 8.3.4 when I made the diff.